include/kernels/mdw_fused_dslash.cuh

```diff
@@ -0,0 +1,451 @@
+#include <gauge_field_order.h>
```

Comment:
SHould that feil be rname to dslash_mdw_fused ?

Reply:
63d7515d2